x-pack/performance/journeys/dashboard_listing_page.ts

```diff
@@ -13,7 +13,20 @@ export const journey = new Journey({
     'x-pack/performance/kbn_archives/flights_no_map_dashboard',
     'x-pack/performance/kbn_archives/logs_no_map_dashboard',
   ],
-}).step('Go to Dashboards Page', async ({ page, kbnUrl }) => {
-  await page.goto(kbnUrl.get(`/app/dashboards`));
-  await page.waitForSelector(`[data-test-subj="table-is-ready"]`);
-});
+})
+  .step('Go to Dashboards Page', async ({ page, kbnUrl }) => {
+    await page.goto(kbnUrl.get(`/app/dashboards`));
+    await page.waitForSelector(`[data-test-subj="table-is-ready"]`);
+  })
+  .step('Search dashboards page', async ({ page, inputDelays }) => {
+    await page.type('[data-test-subj="tableListSearchBox"]', 'Web', {
+      delay: inputDelays.TYPING,
+    });
+    await page.waitForSelector(`[data-test-subj="table-is-ready"]`);
+  })
+  .step('Delete dashboard', async ({ page, log }) => {
+    await page.click('[data-test-subj="checkboxSelectRow-edf84fe0-e1a0-11e7-b6d5-4dc382ef7f5b"]');
+    await page.click('[data-test-subj="deleteSelectedItems"]');
+    await page.click('[data-test-subj="confirmModalConfirmButton"]');
+    await page.waitForSelector(`[data-test-subj="table-is-ready"]`);
```

Comment:
@thomasneirynck, looking at the timings of this journey, it doesn't look like we are waiting for the deletion to complete: `table-is-ready` only changes to `table-is-loading` when refreshing the list after the deletion.

Bearing in mind that the deletion depends on `.kibana`'s `refresh_interval` (set to 1s), we complete the journey before the deletion actually takes place.